### PR TITLE
Add support for ToRhino for NurbsCurve for Rhino7

### LINKED ISSUE #215: Rhinoceros_Engine: Add support for converting BH.oM.Geometry.NurbsCurve ToRhino in Rhino 7
I believe that for the method `ToRhino(this BHG.NurbsCurve bCurve)` all that is required is adding
```
case 7:
    return ToRhino6(bCurve)
```
Edit:
The need for version specific methods for NurbsCurve converts arises due to differences between Rhino 5 and 6. 

Rhino 7 will work with the methods defined in ToRhino6. My tests show the Rhino 5 is the only case where a special method is needed. 

Current code structure suggests this approach will need upgrading with each new version of Rhino but with Rhino 7 this does not appear to be the case. 
I propose to change the code structure so that we have a standard `ToRhino(BH.oM.Geometry.NurbsCurve)` as per all the other convert method. `ToRhino5(BH.oM.Geometry.NurbsCurve)` is only called when the user is running Rhino5. 

For reference @al-fisher + @rwemay + @alelom this issue has come up in pulling and converting Rapid Prototyping model options that use surfaces. The ToRhino method is called when rendering the geometry but also handy for converting back to rhino and leveraging rhino / grasshopper surface modelling tools.

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### Issues addressed by this PR

Closes #215

Changing it so that version is checked and version5 convert is called if version is 5. For versions above 5 the same methodology is used and is instead put in the main method. ToRhino6  method remove and versioned to the main method.

### CODE REVIEW

**Comment by IsakNaslundBh** [comment]
@BHoMBot check compliance

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]
Very minor code formatting changes.

Otherwise looks good on code review - code has moved from the deleted file to the `ToRhino` file which I think it clean and I like the implementation.

**Comment by FraserGreenroyd** [comment]
@BHoMBot check compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required

File: Rhinoceros_Engine/Convert/ToRhino.cs [hunk 1]
```diff
@@ -246,20 +246,40 @@ public static RHG.Line ToRhino(this BHG.Line line)
...
         /***************************************************/
 
+        [PreviousVersion("6.0", "BH.Engine.Rhinoceros.Convert.ToRhino6(BH.oM.Geometry.NurbsCurve)")]
         [Description("Returns the Rhino version of the NurbsCurve.")]
         [Input("bCurve", "Input NurbsCurve.")]
         [Output("rhGeom", "Rhino NurbsCurve.")]
         public static RHG.NurbsCurve ToRhino(this BHG.NurbsCurve bCurve)
         {
-            switch (Rhino.RhinoApp.Version.Major)
+            if (bCurve == null) return null;
```
[the remark this answers is not in the document]
Comment (IsakNaslundBh) [reply Rhinoceros_Engine/Convert/ToRhino.cs]: Fully agree with this. Was a copy paste from the RH6 method. Would have written it as you suggested here.

**Comment by IsakNaslundBh** [comment]
Happy with the changes @FraserGreenroyd .

**Review by FraserGreenroyd — APPROVED** [review]
Should have remembered to do this bit last night...

**Comment by IsakNaslundBh** [comment]
@BHoMBot check ready-to-merge

**Pull request merged**